applications/task-manager/TaskModel.cpp

```diff
@@ -137,3 +138,8 @@ String TaskModel::cpu_greedy()
 {
     return greedy(_data, "cpu");
 }
+
+void TaskModel::kill_task(int row)
+{
+   process_cancel( data(row, 0).as_int());
```

Comment:
why not get the pid directly from _data? also you should do data(row, COLUMN_ID);